src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -196,10 +196,10 @@ private void SetRemoteHostBitness()
                 var checksums = AddGlobalAssets(cancellationToken);
 
                 // send over global asset
-                await client.TryRunRemoteAsync(
-                    WellKnownRemoteHostServices.RemoteHostService, _workspace.CurrentSolution,
-                    nameof(IRemoteHostService.SynchronizeGlobalAssetsAsync),
-                    (object)checksums, cancellationToken).ConfigureAwait(false);
+                await client.IgnoreCancellationAsync(() =>
```

Comment:
Seeing something like IgnoreCancellationAsync seems *extremely strange* to me.

1. either we don't want this to be cancellable, in which case, we probably shouldn't pass cancellation token.
2. we do want this to be cancellable.  But we want to swallow cancellation.  In which case, having an explicit try/catch for that condition seems more appropriate.

Regardless of which approach is taken (including keeping 'IgnoreCancellationAsync'), this sort of complexity really needs to be documented in code.  Code that does not follow normal patterns really needs to have appropriate explanations in it so that people maintaining it down the line know why it's operating in this manner.